app/views/bulk_submissions/index.html.erb

```diff
@@ -29,6 +29,7 @@
               if bulk_submission.pending?
                 button_to(t(".cancel_html", filename:),
                           bulk_submission_path(bulk_submission.id),
+                          class: "govuk-button govuk-!-margin-0 govuk-button--secondary",
```

Comment:
can a govuk component be used? [regular button](https://govuk-components.netlify.app/helpers/button/#regular-button)

Reply:
👍 yes thanks I have amended to use a secondary (grey) button